es-vue-base/src/lib-components/EsFooter.vue

```diff
@@ -15,100 +15,111 @@
-                </div>  <!-- Logo and Social -->
-            </div>
-            <hr class="border-top border-dark m-0">
-            <!-- Trademark Info -->
-            <p class="mt-150 mt-lg-200">
-                {{ content.trademarkText }}
-            </p>
-            <p class="mb-200">
-                {{ copyrightText }}
-            </p>
-            <!-- Trademark Info -->
-            <!-- Legal -->
-            <div class="row justify-content-between mb-150">
-                <div
-                    v-for="link in content.legalLinks"
-                    :key="link.text"
-                    class="col col-6 col-lg font-size-50 mt-25 mb-50">
-                    <es-nav-bar-link
-                        :href="link.url"
-                        class="text-reset font-weight-normal">
-                        {{ link.text }}
-                    </es-nav-bar-link>
+                        <!-- Social icons -->
+                        <div class="d-flex text-white">
+                            <es-nav-bar-link
+                                v-for="iconLink in content.socialLinks"
+                                :key="iconLink.text"
+                                class="text-reset pr-150"
+                                :href="iconLink.url"
+                                target="_blank">
+                                <span class="sr-only">{{ iconLink.text }}</span>
+                                <component :is="iconLink.icon" />
+                            </es-nav-bar-link>
+                        </div>  <!-- Social icons -->
+                    </div>  <!-- Logo and Social -->
                 </div>
-            </div>  <!-- Legal -->
-            <!-- DOE -->
-            <div class="d-lg-flex align-items-end">
-                <!-- eslint-disable-next-line -->
-                <img
-                    :src="content.departmentOfEnergy.logoUrl"
-                    alt="department of energy logo"
-                    aria-hidden="true"
-                    class="mr-100 mb-100 mb-lg-0"
-                    width="99"
-                    height="25">
-                <p class="mb-0">
-                    <es-nav-bar-link
-                        :href="content.departmentOfEnergy.learnMore.link"
-                        target="_blank">
-                        {{ content.departmentOfEnergy.learnMore.text }}
-                    </es-nav-bar-link>
+            </div>
+        </div>
+        <div class="bg-blue-700 text-blue-50 px-200 py-100">
+            <div class="container px-0">
+                <!-- Trademark Info -->
+                <p class="font-size-75">
+                    {{ content.trademarkText }}
+                </p>
+                <p class="mb-200 font-size-75">
+                    {{ copyrightText }}
                 </p>
-            </div>  <!-- DOE -->
+                <!-- Trademark Info -->
+                <!-- Legal -->
+                <hr class="border-top bg-blue-50 m-0">
```

Comment:
`bg-blue-50` doesn't do anything to an `<hr>` element as the only visible part is the border. You probably want `border-blue-50` but there is no such class right now.

I'd be okay with you adding a loop to the blue colors to `/scss/utilities/_borders.scss` to generate a border color class for the list of blues:
```
@each $color, $value in variables.$blues {
  .border-#{$color} {
    border-color: $value !important;
  }
}
```

Then you can use a `border-blue-50` class here.

Reply:
Oops I misspoke on the correct color name - check the Figma, I think it may need to be `border-blue-500`.